src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/ExtendableClientCapabilities.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using OmniSharp.Extensions.LanguageServer.Protocol.Client.Capabilities;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer
+{
+    internal class ExtendableClientCapabilities : ClientCapabilities
```

Comment:
Support for custom Client Capabilities aside from what the base LSP spec offers.

Reply:
I might leave a comment on this class stating that any values set here will immediately impact all LSP interactions. Also might be valuable to leave a similar comment where we register the JsonConverter for this so readers know the implications

Reply:
I'm not sure I follow. How would this impact all LSP interactions? This is only set on initialization of the omnisharp language server, and _is_ still a `ClientCapabilities`.

Reply:
omg forget me. I was confusing it with server capabilities. Neverrrmind!